storage/test/test_kv_leveldb_get_equal_or_prev.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from storage.kv_store_leveldb_int_keys import KeyValueStorageLeveldbIntKeys
+
+
+@pytest.fixture(scope="function")
+def storage_with_ts_root_hashes(tmpdir):
+    storage = KeyValueStorageLeveldbIntKeys(tmpdir.dirname, "test_db")
+    ts_list = {
+        2: "aaaa",
+        4: "bbbb",
+        5: "cccc",
+        100: "ffff",
+    }
+    for k, v in ts_list.items():
+        storage.put(str(k), v)
+    return storage, ts_list
+
+
+def test_none_if_key_less_then_minimal_key(storage_with_ts_root_hashes):
```

Comment:
We may consider having the same tests for RocksDB implementation as well (adding parameters to the tests), but I think it can be done with RocksDB support.